<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>987</width>
    <height>681</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>987</width>
    <height>677</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>987</width>
    <height>681</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>账号管理</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>C:/Users/Lenovo/.designer/res_img/群蜂账号管理.png</normaloff>C:/Users/Lenovo/.designer/res_img/群蜂账号管理.png</iconset>
  </property>
  <property name="windowFilePath">
   <string/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>991</width>
     <height>681</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>991</width>
     <height>681</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>C:/Users/Lenovo/.designer/res_img/背景.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>-10</y>
     <width>911</width>
     <height>836</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabWidget::pane{
min-width:70px;
min-height:25px;
border-top: 2px solid;
}
QTabWidget::tab-bar{padding-top:40px;}
QTabBar::tab {
min-width: 140px;
margin: 10px; 

min-width:70px;

min-height:25px;

color: rgb(50,50,50);

font:20px &quot;华文行楷&quot;;

border: 0px solid;



}

QTabBar::tab:selected{

min-width:70px;

min-height:25px;
color: rgb(0,0,0);

font:20px &quot;华文行楷&quot;;

border: 0px solid;

border-bottom: 2px solid;

border-color: black;

}</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>已有账号</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget_15">
     <property name="geometry">
      <rect>
       <x>31</x>
       <y>85</y>
       <width>861</width>
       <height>331</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
selection-color: rgb(79, 80, 97);
selection-background-color:rgba(0, 0, 0, 100);;
border-color: transparent;
font: 16pt &quot;华文行楷&quot;;</string>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>账号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>姓名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>性别</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>联系方式</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>备注</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>操作</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="btn_seek_user">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>30</y>
       <width>71</width>
       <height>45</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>45</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
    font: 18pt &quot;华文行楷&quot;;
    background: rgba(255, 255, 255, 60);
    color: black;
    border-radius: 16px;
    font-family: 华文行楷;
    border: 2px solid rgb(100,100,100); /* 新增代码 */
}

QPushButton:hover{                    
    background: rgba(0, 0, 0, 100);
}

QPushButton:pressed{
    background: darkgray;
}</string>
     </property>
     <property name="text">
      <string>查询</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_updata_user">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>30</y>
       <width>71</width>
       <height>45</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>45</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
    font: 18pt &quot;华文行楷&quot;;
    background: rgba(255, 255, 255, 60);
    color: black;
    border-radius: 16px;
    font-family: 华文行楷;
    border: 2px solid rgb(100,100,100); /* 新增代码 */
}

QPushButton:hover{                    
    background: rgba(0, 0, 0, 100);
}

QPushButton:pressed{
    background: darkgray;
}</string>
     </property>
     <property name="text">
      <string>更新</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>351</width>
       <height>47</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_69">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>华文行楷</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>关键词：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_48">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>45</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>华文行楷</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    border: 2px solid rgb(130,130,130); /* 设置初始边框为浅白色 */
    border-radius: 10px; /* 设置圆角半径为10px，可根据需要调整 */
}

QLineEdit:hover {
    border: 4px solid #606060; /* 鼠标悬停时边框变粗且变为浅黑色 */
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>账号/姓名/联系方式</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>添加账号</string>
    </attribute>
    <widget class="QLabel" name="label_76">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>120</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>华文行楷</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>添加账号</string>
     </property>
    </widget>
    <widget class="Line" name="line_64">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>58</y>
       <width>901</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>8</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_65">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>400</y>
       <width>911</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>8</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>100</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文行楷</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>男</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>100</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文行楷</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>女</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_74">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>100</y>
       <width>60</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>华文行楷</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>性别</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_add_user">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>340</y>
       <width>71</width>
       <height>45</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>45</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
    font: 18pt &quot;华文行楷&quot;;
    background: rgba(255, 255, 255, 60);
    color: black;
    border-radius: 16px;
    font-family: 华文行楷;
    border: 2px solid rgb(100,100,100); /* 新增代码 */
}

QPushButton:hover{                    
    background: rgba(0, 0, 0, 100);
}

QPushButton:pressed{
    background: darkgray;
}</string>
     </property>
     <property name="text">
      <string>添加</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_clear_user">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>340</y>
       <width>71</width>
       <height>45</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>45</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
    font: 18pt &quot;华文行楷&quot;;
    background: rgba(255, 255, 255, 60);
    color: black;
    border-radius: 16px;
    font-family: 华文行楷;
    border: 2px solid rgb(100,100,100); /* 新增代码 */
}

QPushButton:hover{                    
    background: rgba(0, 0, 0, 100);
}

QPushButton:pressed{
    background: darkgray;
}</string>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>139</width>
       <height>291</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,1,1,1">
      <item>
       <widget class="QLabel" name="label_70">
        <property name="font">
         <font>
          <family>华文行楷</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>姓        名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_73">
        <property name="font">
         <font>
          <family>华文行楷</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>联系方式 </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>华文行楷</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>备        注</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_77">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>45</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>华文行楷</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>账        号</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_78">
        <property name="font">
         <font>
          <family>华文行楷</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>密        码   </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>90</y>
       <width>351</width>
       <height>291</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1,1,1,1">
      <item>
       <widget class="QLineEdit" name="lineEdit_49">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>45</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    border: 2px solid rgb(130,130,130); /* 设置初始边框为浅白色 */
    border-radius: 10px; /* 设置圆角半径为10px，可根据需要调整 */
	font: 16pt &quot;华文行楷&quot;;
}

QLineEdit:hover {
    border: 4px solid #606060; /* 鼠标悬停时边框变粗且变为浅黑色 */
}</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_52">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>45</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    border: 2px solid rgb(130,130,130); /* 设置初始边框为浅白色 */
    border-radius: 10px; /* 设置圆角半径为10px，可根据需要调整 */
	font: 16pt &quot;华文行楷&quot;;
}

QLineEdit:hover {
    border: 4px solid #606060; /* 鼠标悬停时边框变粗且变为浅黑色 */
}</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_54">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>45</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    border: 2px solid rgb(130,130,130); /* 设置初始边框为浅白色 */
    border-radius: 10px; /* 设置圆角半径为10px，可根据需要调整 */
	font: 16pt &quot;华文行楷&quot;;
}

QLineEdit:hover {
    border: 4px solid #606060; /* 鼠标悬停时边框变粗且变为浅黑色 */
}</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_55">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>45</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    border: 2px solid rgb(130,130,130); /* 设置初始边框为浅白色 */
    border-radius: 10px; /* 设置圆角半径为10px，可根据需要调整 */
	font: 16pt &quot;华文行楷&quot;;
}

QLineEdit:hover {
    border: 4px solid #606060; /* 鼠标悬停时边框变粗且变为浅黑色 */
}</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_56">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>45</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    border: 2px solid rgb(130,130,130); /* 设置初始边框为浅白色 */
    border-radius: 10px; /* 设置圆角半径为10px，可根据需要调整 */
	font: 16pt &quot;华文行楷&quot;;
}

QLineEdit:hover {
    border: 4px solid #606060; /* 鼠标悬停时边框变粗且变为浅黑色 */
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_79">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>550</y>
       <width>241</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>华文行楷</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>管理员密码修改：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>550</y>
       <width>321</width>
       <height>45</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    border: 2px solid rgb(130,130,130); /* 设置初始边框为浅白色 */
    border-radius: 10px; /* 设置圆角半径为10px，可根据需要调整 */
	font: 16pt &quot;华文行楷&quot;;
}

QLineEdit:hover {
    border: 4px solid #606060; /* 鼠标悬停时边框变粗且变为浅黑色 */
}</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>550</y>
       <width>81</width>
       <height>45</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
    font: 18pt &quot;华文行楷&quot;;
    background: rgba(255, 255, 255, 60);
    color: black;
    border-radius: 16px;
    font-family: 华文行楷;
    border: 2px solid rgb(100,100,100); /* 新增代码 */
}

QPushButton:hover{                    
    background: rgba(0, 0, 0, 100);
}

QPushButton:pressed{
    background: darkgray;
}</string>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_80">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>440</y>
       <width>301</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>华文行楷</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>管理员密码修改查看：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>440</y>
       <width>321</width>
       <height>45</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    border: 2px solid rgb(130,130,130); /* 设置初始边框为浅白色 */
    border-radius: 10px; /* 设置圆角半径为10px，可根据需要调整 */
	font: 16pt &quot;华文行楷&quot;;
}

QLineEdit:hover {
    border: 4px solid #606060; /* 鼠标悬停时边框变粗且变为浅黑色 */
}</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>440</y>
       <width>81</width>
       <height>45</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
    font: 18pt &quot;华文行楷&quot;;
    background: rgba(255, 255, 255, 60);
    color: black;
    border-radius: 16px;
    font-family: 华文行楷;
    border: 2px solid rgb(100,100,100); /* 新增代码 */
}

QPushButton:hover{                    
    background: rgba(0, 0, 0, 100);
}

QPushButton:pressed{
    background: darkgray;
}</string>
     </property>
     <property name="text">
      <string>查看</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_exit">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>10</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    font: 18pt &quot;华文行楷&quot;;
    background: rgba(255, 255, 255, 60);
    color: black;
    border-radius: 16px;
    font-family: 华文行楷;
}

QPushButton:hover{                    
    background: rgba(0, 0, 0, 100);
}

QPushButton:pressed{
    background: darkgray;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/Lenovo/.designer/res_img/退出.png</normaloff>C:/Users/Lenovo/.designer/res_img/退出.png</iconset>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
